pkg/apis/plugin/plugin.go

```diff
@@ -54,6 +54,7 @@ type UserDataRequest struct {
 	KubeletRepository     string
 	KubeletFeatureGates   map[string]bool
 	ContainerRuntime      containerruntime.Config
+	UseTrustedCA          bool
```

Comment:
Seems like this field is not used anymore

Reply:
Yes, I'd remove that